gen/elerning_mu_ff

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/perl
```

Comment:
I think there is missing `a` in the service name, as it should be "elearning". Technically it works, since the service name is the same everywhere, but maybe discuss it with Michal if he wants to fix it.

Reply:
It should be e-learning but it is in so many places at this moment that I would let it for a further fix.